test/drivers/test_cqrrp.cc

```diff
@@ -167,7 +168,37 @@ TEST_F(TestCQRRP, CQRRP_blocked_full_rank_basic) {
     test_CQRRP_general(d_factor, norm_A, all_data, CQRRP_blocked, state);
 #endif
 }
+/*
+// Note: If Subprocess killed exception -> reload vscode
+TEST_F(TestCQRRP, CQRRP_blocked_full_rank_custom_070824_cpu) {
+    int64_t m = 5000;//5000;
+    int64_t n = 2000;//2000;
+    int64_t k = 2000;
+    double d_factor = 1;//1.0;
+    int64_t b_sz = 500;//500;
+    double norm_A = 0;
+    double tol = std::pow(std::numeric_limits<double>::epsilon(), 0.85);
+    auto state = RandBLAS::RNGState();
```

Comment:
Why commented out?

Reply:
No idea tbh, but this file would be fully changed after a rebase anyway.